regulations/static/regulations/js/source/views/main/search-results-view.js

```diff
@@ -78,30 +83,32 @@ var SearchResultsView = ChildView.extend({
     paginate: function(e) {
         e.preventDefault();
 
-        var page = $(e.target).hasClass('previous') ? this.page - 1 : this.page + 1,
-            config = {
-                query: this.query,
-                regVersion: this.resultsRegVersion,
-                page: page
-            };
+        var options = {
+          query: this.options.query,
+          docType: this.options.docType,
+          regVersion: this.options.regVersion,
+          page: this.page + ($(e.target).hasClass('previous') ? -1 : 1)
+        };
 
-        MainEvents.trigger('search-results:open', null, config, 'search-results');
+        MainEvents.trigger('search-results:open', null, options, 'search-results');
     },
 
     openResult: function(e) {
         // TOC version retains the version the reg was loaded on whereas the content base section
         // changes to match the search results
         // page should reload if the TOC version doesn't match the searched version
-        if (this.resultsRegVersion === $('nav#toc').attr('data-toc-version')) {
+        if (!this.resultsRegVersion || this.resultsRegVersion === $('nav#toc').attr('data-toc-version')) {
```

Comment:
Is `this.resultsRegVersion` only set for `cfr` docs? If so, I think this guard would be easier to follow if we validated the docType directly.